.github/workflows/php-pr-checks.yml

```diff
@@ -0,0 +1,58 @@
+name: PHPUnit / PHPCS / Phan
+on:
+  pull_request:
+    branches: '**'
+
+  push:
+    branches: [ master, MW_1_37, REL1_39 ]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        php_version: ['8.0']
+    runs-on: ubuntu-latest
+    steps:
+    - name: Setup PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: ${{ matrix.php_version }}
+        # We don't run any coverage, so we should set this parameter to none
+        # as this will disable xdebug which slows all processes significantly
+        coverage: none
+        extensions: ast
+    - uses: actions/checkout@v3
+
+    - name: Cache Composer packages
+      id: composer-cache
+      uses: actions/cache@v3
+      with:
+        path: vendor
+        key: ${{ runner.os }}-php-v3-${{ hashFiles('**/composer.lock') }}
+
+    - name: Install composer dependencies
+      if: steps.composer-cache.outputs.cache-hit != 'true'
+      run: composer install --prefer-dist --no-progress
+
+    - name: Cache npm packages
+      id: npm-cache
+      uses: actions/cache@v3
+      with:
+        path: vendor
+        key: ${{ runner.os }}-npm-v3-${{ hashFiles('**/package-lock.json') }}
+
+    - name: Install npm dependencies
+      if: steps.npm-cache.outputs.cache-hit != 'true'
+      run: npm install
+
+    - name: Run PHPCS
+      run: composer phpcs
+
+    - name: Run Phan static analysis
+      run: composer phan
+
+    - name: Run PHPUnit tests with coverage
+      run: composer phpunit
+
+    - name: Run Jasmine tests
+      run: npm test
```

Comment:
It's basically `assertTrue` test, but let's leave integration with Jasmine ready
https://github.com/Wikia/reverb-ext/blob/230f1da0269cdc2abf30aaa9f27cadfcca7f1769/spec/jasmine_examples/ExampleSpec.js#L1-L8

Reply:
Let's reflect the fact we're running JS tests in the check name too